cli/command/image/build/context.go

```diff
@@ -242,9 +242,9 @@ func getWithStatusError(url string) (resp *http.Response, err error) {
 	body, err := ioutil.ReadAll(resp.Body)
 	resp.Body.Close()
 	if err != nil {
-		return nil, errors.Wrapf(err, msg+": error reading body")
+		return nil, errors.Wrapf(err, fmt.Sprintf("%s: error reading body", msg))
```

Comment:
I think instead of this, this should be `errors.Wrap()` instead of `errors.Wrapf()`;

```suggestion
		return nil, errors.Wrap(err, msg+": error reading body")
```